atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/StreamStoreRemappingSweepPriorityCalculatorTest.java

```diff
@@ -0,0 +1,634 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.sweep;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.schema.stream.StreamTableType;
+import com.palantir.atlasdb.sweep.priority.ImmutableSweepPriority;
+import com.palantir.atlasdb.sweep.priority.StreamStoreRemappingSweepPriorityCalculator;
+import com.palantir.atlasdb.sweep.priority.SweepPriority;
+import com.palantir.atlasdb.sweep.priority.SweepPriorityCalculator;
+import com.palantir.atlasdb.sweep.priority.SweepPriorityStore;
+
+public class StreamStoreRemappingSweepPriorityCalculatorTest {
```

Comment:
This test is pretty solid, and covers quite a lot of interesting cases (e.g. `standardEstimatedTablePriorities`, which makes good sense). 👍